atd-etl/cr3_extract_diagram/cr3_extract_diagram_ocr_narrative.py

```diff
@@ -164,7 +164,7 @@ def update_crash_metadata(crash_id: int, metadata: dict) -> bool:
   atd_txdot_crashes(where: {
       cr3_ocr_extraction_date: {_is_null: true},
       crash_id: {_gt: 10000}
-      crash_date: {_gte: "2015-01-01"}
+      crash_date: {_gte: "2020-01-01"}
```

Comment:
whats going on here?

Reply:
hmm—i think i either branched from the wrong place or merged some upstream commits along the way. these changes are already in `master` via https://github.com/cityofaustin/atd-vz-data/pull/1469/commits. this should not be a big deal 🤷